File: add/codesnippet/CPP/e4d60361-8137-4437-9775-_1.cpp
   CspParameters^ parameters = gcnew CspParameters;
   parameters->KeyContainerName = L"TestContainer";
   array<Object^>^argsArray = gcnew array<Object^>(1){
      parameters
   };
   
   // Instantiate the RSA provider instance accessing the TestContainer
   // key container.
   RSACryptoServiceProvider^ rsaProvider =
      static_cast<RSACryptoServiceProvider^>(
         CryptoConfig::CreateFromName( L"RSA", argsArray ));

File: add/codesnippet/CSharp/e4d60361-8137-4437-9775-_1.cs
        CspParameters parameters = new CspParameters();
        parameters.KeyContainerName = "TestContainer";
        Object[] argsArray = new Object[] {parameters};

        // Instantiate the RSA provider instance accessing the TestContainer
        // key container.
        RSACryptoServiceProvider rsaProvider = (RSACryptoServiceProvider) 
            CryptoConfig.CreateFromName("RSA",argsArray);